klog.go

```diff
@@ -803,6 +803,16 @@ func SetOutputBySeverity(name string, w io.Writer) {
 	logging.file[sev] = rb
 }
 
+// LogToOutput configures klog to output to the specified writer. It is equivalent to SetOutput()
+// and flag.Set("logtostderr", "false")
+func LogToOutput(w io.Writer) {
+	SetOutput(w)
+	logging.mu.Lock()
+	defer logging.mu.Unlock()
+
+	logging.toStderr = false
```

Comment:
If we wanted to give programmatic access to `toStderr`, that seems more reasonable. A single method that wraps a call to an existing method seems fairly inflexible.

Reply:
That's reasonable, I can knock out a PR like that real quit